io-engine/src/bdev/nexus/nexus_io.rs

```diff
@@ -253,10 +266,20 @@ impl<'n> NexusBio<'n> {
         if status == IoCompletionStatus::Success {
             self.ctx_mut().successful += 1;
         } else {
+            error!(
```

Comment:
we already have traces below? I'd prefer not to add too many traces in the IO path until we add a redirection. Currently traces are a blocking call so if stdout/err is not pulled fast enough it can block the task.